packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -90,14 +91,9 @@ class _DropdownMenuPainter extends CustomPainter {
 
 // Do not use the platform-specific default scroll configuration.
 // Dropdown menus should never overscroll or display an overscroll indicator.
-class _DropdownScrollBehavior extends ScrollBehavior {
-  const _DropdownScrollBehavior();
-
-  @override
-  TargetPlatform getPlatform(BuildContext context) => Theme.of(context).platform;
-
-  @override
-  Widget buildViewportChrome(BuildContext context, Widget child, AxisDirection axisDirection) => child;
+// The default scrollbar platforms will apply.
+class _DropdownScrollBehavior extends MaterialScrollBehavior {
```

Comment:
If you have a custom ScrollConfiguration higher in the tree that builds custom non-material scrollbars, this widget would have unmatching material scrollbars...

Reply:
Presumably, we want to use the copy-with approach here as well...